src/style.js

```diff
@@ -297,6 +297,16 @@ export function* groupIndex(I, position, mark, channels) {
   }
 }
 
+// TODO Accept other types of clips (paths, urls, x, y, other marks…)?
+// https://github.com/observablehq/plot/issues/181
+export function maybeClip(clip) {
+  if (clip === true) clip = "frame";
+  else if (clip === false) clip = null;
+  else if (clip != null) clip = keyword(clip, "clip", ["frame", "sphere"]);
+  return clip;
+}
```

Comment:
This doesn’t look used. Is this needed as part of this PR?

Reply:
Good catch. This was the result of a bad merge operation (this part belongs to #1624).